### ISSUE TITLE
DataInspector tooltips swallow punctuation on first plot

### ISSUE THREAD

**Issue by staticfloat** [MakieOrg/Makie.jl#3232]
Using `WGLMakie#sd/beta-20`, on a Macbook Pro M1.  The following long DataInspector tooltip:

```
julia> using WGLMakie
       f = Figure()
       ax = Axis(f[1,1])
       scatter!(ax, [1, 2, 3], [1, 2, 1]; inspector_label = (args...) -> "This is a really long tooltip\nI'm trying to trigger the issue Glen mentions\nMaybe I can do it if I keep on typing\nBut this length seems to work\nwhat's the issue? I can't find it!")
       DataInspector(f)
       f
```

Results in a tooltip without the `?` or `!`:

![image](https://github.com/MakieOrg/Makie.jl/assets/130920/2490336e-e4dd-488a-80d2-68a1997a086f)

But when I run the exact same code again a second time, the glyphs properly render.

**Comment by jkrumbiegel** [comment]
I think those glyphs are not in the default set of glyphs in the atlas, maybe there's a race condition when you plot them for the first time.

**Comment by ffreyer** [comment]
Not sure why github didn't close this but the problem should be fixed in 0.20.9
